docs/api.md

```diff
@@ -201,6 +201,32 @@ Default: `'msg'`
 
 The string key for the 'message' in the JSON object.
 
+<a id=opt-objectkey></a>
+#### `objectKey` (String)
```

Comment:
`objectKey` is somewhat vague, can we change the name to `nestedKey`

Reply:
renamed